test/index.js

```diff
@@ -302,5 +303,26 @@ describe('big-json', function() {
                 return done();
             });
         });
+
+        it('should parse (buffer)', function(done) {
```

Comment:
Nit - can this be `should parse buffer (promise)` just to keep consistency with the tests above?